documentation/development.md

```diff
@@ -0,0 +1,55 @@
+# Git Town Development
+
+## Requirements
+
+* Ruby 2.1.5
+  (install [directly](https://www.ruby-lang.org/en/documentation/installation/)
+   or with [rvm](https://rvm.io/))
+* [ShellCheck](https://github.com/koalaman/shellcheck)
```

Comment:
homebrew is mac specific. If you go to the shellcheck webpage it gives you all the different ways you can install it